src/components/pages/DashboardPage/TotalLOCs.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import {connect} from 'react-redux'
+
+import ThumbUp from 'material-ui/svg-icons/action/thumb-up'
+import InfoBox from './InfoBox'
+import {getLOCs} from '../../../redux/locs/actions'
+import {CircularProgress} from 'material-ui'
+
+const mapStateToProps = (state) => ({
+  locs: state.get('locs'),
+  isReady: state.get('locsCommunication').isReady,
+  isFetching: state.get('locsCommunication').isFetching
+})
+
+const mapDispatchToProps = (dispatch) => ({
+  getLOCs: (account) => dispatch(getLOCs(account))
+})
+
+@connect(mapStateToProps, mapDispatchToProps)
+class TotalLOCs extends Component {
+  render () {
+    return (
+      <div>
+        <InfoBox Icon={ThumbUp}
+          color='#17579c'
+          title='LOCs'
+          value={this.props.isFetching
+            ? <CircularProgress size={24} thickness={1.5} style={{float: 'right'}} />
+            : <span>{this.props.locs.size}</span>
```

Comment:
this is unsuitable decision, because we're going to implement pagination to LOCs page in the future
you should somehow get number of LOCs directly from blockchain